validator/src/ethereum/fees.ts

```diff
@@ -4,18 +4,22 @@ import type { GasFee } from "./types.js";
 export const getGasFee = async (
 	harbourClient: Client,
 	basePriceMultiplier?: bigint,
+	priorityFeeMultiplier?: bigint,
 ): Promise<GasFee> => {
 	const feeHistory = await harbourClient.request({
 		method: "eth_feeHistory",
 		params: ["0x1", "latest", [100]],
 	});
-	const maxPriorityFeePerGas = await harbourClient.request({
-		method: "eth_maxPriorityFeePerGas",
-	});
+	const maxPriorityFeePerGas =
+		hexToBigInt(
+			await harbourClient.request({
+				method: "eth_maxPriorityFeePerGas",
+			}),
+		) * (priorityFeeMultiplier ?? 2n);
```

Comment:
Pimlico is complaining on Gnosis Chain that the priority fee is too low, therefore we double it. This needs to be refactored.